test/Concurrency/async_task_groups.swift

```diff
@@ -33,11 +33,11 @@ func test_taskGroup_add() async throws -> Int {
 
 func test_taskGroup_addHandles() async throws -> Int {
   await try Task.withGroup(resultType: Int.self) { group in
-    let one = await group.addWithHandle {
+    let one = await group.add {
```

Comment:
`addWithHandle` is gone, it is not necessary, we simply return the handle always and it is discardable